src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -299,6 +299,22 @@ private String getCurrentDateAndTimeFormatted() {
         return dateFormat.format(new Date());
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) This class overrides "equals()" and should therefore also override "hashCode()". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1206)